ppa/archive/gale.py

```diff
@@ -192,14 +208,19 @@ def get_item_pages(self, item_id, gale_record=None):
         # iterate through the pages in the response
         for page in gale_record["pageResponse"]["pages"]:
             page_number = page["pageNumber"]
-            # page label (original page number) should be set in folioNumber,
-            # but is not set for all volumes; fallback to page number
-            # converted to integer to drop leading zeroes
-            page_label = page.get("folioNumber", int(page_number))
+            tags = []
```

Comment:
suggest adding a comment here with a bit more details

```suggestion
            tags = []
            # get higher quality local OCR text content if possible, with fallback to Gale OCR
            # set a tag so we have some way to determine when local OCR is present
            
```